### PR TITLE
Fix GH fork check

### PR DESCRIPTION
**Pull request by pawelprazak**
We seem to be ignoring branch pushes by accident. I hope this will solve the issue, if not we need to dig a little deeper into this.

### CODE REVIEW

**Review by KacperFKorban — APPROVED** [review]

**Comment by lbialy** [comment]
@pawelprazak this is actually redundant either way because gha does not execute pipelines on PRs from forks, it asks for a permission to run them. CI files are also in CODEOWNERS so can't be changed without explicit approval from the team.

**Pull request merged**
